### PR TITLE
inadyn: update to 2.7

### PR DESCRIPTION
**Pull request by neheb**
Remove pic hack.

Fix license information.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @openwrt/package-maintainers 
Compile tested: ath79

### CODE REVIEW

File: net/inadyn/Makefile [hunk 1]
```diff
@@ -41,16 +41,13 @@ define Package/inadyn/conffiles
...
 endef
 
-TARGET_CFLAGS += $(FPIC)
-
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--disable-static \
 	--enable-openssl \
-	--without-pic
+	--with-pic
```
Comment (aparcar) [inline net/inadyn/Makefile:48]: Why this change?

Comment (neheb) [reply net/inadyn/Makefile:48]: Removed the CFLAGS hack and replaced with this.
